packages/bot/src/stores/journal-store.js

```diff
@@ -13,7 +17,16 @@ export default class JournalStore {
         return this.root_store.core.common.server_time;
     }
 
+    filter_list = [
+        { id: message_types.ERROR, label: 'Error messages', onChange: this.filterMessage },
```

Comment:
- Please use `localize` for labels.
- Could we simplify the labels to `Errors`, `Notifications`, and `System`?

Reply:
> Could we simplify the labels to Errors, Notifications, and System
Sure, but let me confirm with copywriters first